### DISCUSSION TITLE
Conventional Commits

### DISCUSSION THREAD

**Discussion by 0x4007** [pavlovcik/uad-ui#423] [Research]
> [react: 5329] https://github.com/reactjs/reactjs.org/pull/5329/commits
> [next: 44163] https://github.com/vercel/next.js/pull/44163/commits
> [oz: 3861] https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3861/commits

- https://github.com/facebook/react/commits/main?after=5fcf1a4b4c2150a1b9fe0de0144a82a053c63966+34&branch=main&qualified_name=refs%2Fheads%2Fmain

- https://github.com/vercel/next.js/commits/canary

- https://github.com/OpenZeppelin/openzeppelin-contracts/commits/master

So you bring up a good point @Steveantor but it looks like they are using some other convention with the # references within the commit messages 🤔

_Originally posted by @pavlovcik in https://github.com/ubiquity/ubiquity-dollar/issues/407#issuecomment-1362949358_

**Comment by 0x4007** [comment]
@Steveantor I just moved the conversation outside of the PR and into here in order to minimize polluting not related topics.

**Reply by Steveantor** [reply]
> > [react: 5329] https://github.com/reactjs/reactjs.org/pull/5329/commits
> > [next: 44163] https://github.com/vercel/next.js/pull/44163/commits
> > [oz: 3861] https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3861/commits
> 
> * https://github.com/facebook/react/commits/main?after=5fcf1a4b4c2150a1b9fe0de0144a82a053c63966+34&branch=main&qualified_name=refs%2Fheads%2Fmain
> * https://github.com/vercel/next.js/commits/canary
> * https://github.com/OpenZeppelin/openzeppelin-contracts/commits/master
> 
> So you bring up a good point @Steveantor but it looks like they are using some other convention with the # references within the commit messages 🤔
> 
> _Originally posted by @pavlovcik in [#407 (comment)](https://github.com/ubiquity/ubiquity-dollar/pull/407#issuecomment-1362949358)_

React probably using a custom artifact.
